crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/MaatCourtDataAssessmentBuilder.java

```diff
@@ -30,8 +30,13 @@ public MaatApiAssessmentRequest build(final MeansAssessmentDTO assessment, final
 
         MaatApiAssessmentRequest apiAssessmentRequest;
 
-        if (requestType.equals(AssessmentRequestType.UPDATE)) {
+        if (AssessmentRequestType.UPDATE.equals(requestType)) {
```

Comment:
This change means that child weightings will no longer be present when creating an INIT assessment, only when updating them.

The condition for whether to include the child weightings should be as follows:

`if (AssessmentType.INIT.equals(assessmentType)`

The AssessmentRequestType CREATE/UPDATE doesn't matter, just the assessment type.

I would roll back the latest changes and add a separate if condition below here.

Reply:
I believe the acceptance criteria in the Jira ticket is confusing: